project-communications/newsletter/newsletter-June21.md

```diff
@@ -0,0 +1,122 @@
+# Newsletter - June 2021
+
+**Hi everyone!!**
+
+Firstly, thank you for opening and reading this month's newsletter 😄 We are so happy you are here!
+
+June is showing us some warmer weather that I really hope is raising spirits! 
+
+There is a lot of change happening right now with the COVID roadmap having come out a couple of months ago and things starting to open up. 
+If anyone is struggling with these challenging times, we hope that the citizen science community can offer your support and connection.
+
+Email [Lotty](mailto:ccoupat@turing.ac.uk) to let us know if we can help you with involvement, and do feel free to share this newsletter to anyone who you think would be interested in joining us.
+
+They can sign up by following this link: https://tinyletter.com/AutisticaTuringCitizenScience.
+
+Best wishes for everyone! 💮
+
+# Project Update
+
+The most recent updates on the project and it's progress can be found [here](https://docs.google.com/presentation/d/1gW2twVD52a5MCjm_MrHdi464YHNFluNyPMWN-96lOjM/edit?usp=sharing).
+
+A lot of our resource has been going towards roadmapping the project and ensuring all our issues are up to date! 
+Our latest roadmap can be found in our github repository [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-management/project-roadmap).
+
+## Contributor Spotlight
+
+Today we are spotlighting on Anoushka Ramesh who has been a key contributor to the AutSPACEs platform development! 
+
+**Tell us a little about yourself...**
+
+Hi everyone! 
+I’m Anoushka Ramesh, a Computer Science undergraduate from India and an active contributor to AutSPACEs. 
+Interestingly enough, my love for Computer Science really grew during the pandemic where I first started delving into mini projects related to frontend technologies like HTML, CSS and slowly made my way to learning APIs and other tools like Python, React and Django. 
+The open source path looked like a perfect opportunity to apply my new found knowledge for a good cause. AutSPACEs caught my eye as I was going through the list of INCF open source projects for the Google Summer of Code program. 
+It sounded like a wonderful project that aimed to dig deep into understanding the causes behind autism. 
```

Comment:
the experiences of autistic people?

Reply:
Oh yes, that should be rectified. @lottycoupat could you change that and add 'experiences of autistic people and what causes environments to be better or worse for autistic people.' instead of 'causes behind autism'?

Reply:
@anoura12 @GeorgiaHCA amendments made!

Reply:
Fantastic work both!